web/js/layoutManager.js

```diff
@@ -35,6 +35,42 @@ LayoutViewport, ItemsHandler */
     }
   };
 
+  var throttle = function(func, wait, options) {
+           var context, args, result;
+           var timeout = null;
+           var previous = 0;
+           if (!options) options = {};
+           var later = function() {
+             previous = options.leading === false ? 0 : _.now();
+             timeout = null;
+             result = func.apply(context, args);
+             if (!timeout) context = args = null;
+           };
+           return function() {
+             var now = _.now();
+             if (!previous && options.leading === false) previous = now;
+             var remaining = wait - (now - previous);
+             context = this;
+             args = arguments;
+             if (remaining <= 0 || remaining > wait) {
+               clearTimeout(timeout);
+               timeout = null;
+               previous = now;
+               result = func.apply(context, args);
+               if (!timeout) context = args = null;
+             } else if (!timeout && options.trailing !== false) {
+               timeout = setTimeout(later, remaining);
+             }
+             return result;
+           };
+     };
+
+   var adjustLayout = function(event) {
+        console.log(getTotal()+" the orientation of the device is now " + event.target.screen.orientation.angle);
```

Comment:
Please remove the call to `console.log()`. Can we delete the `adjustLayout()` function?

Reply:
yes cleaning this

Reply:
This is an outdated file. the latest commit doesn't have this code.